### PR TITLE
Completed queries: 2, 3, 4, 5, 6, 8, 9

### PR DESCRIPTION
**Pull request by Michi18**

### CODE REVIEW

File: Round_1_query_4.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SELECT
+host_id,
+COUNT(neighbourhood) as count_neighbourhood
```
Comment (pmyrsi) [inline Round_1_query_4.sql]: Think about `neighbourhood` column - does it really guarantee location uniqueness?

File: Round_1_query_6.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+SELECT
+COUNT(listing_id)
```
Comment (pmyrsi) [inline Round_1_query_6.sql]: You're counting these listings correctly, however the task is to calculate the % of listings with WiFi - seems this is incomplete still :)

File: Round_1_query_4.sql (hunk 1 above)
Comment (pmyrsi) [inline Round_1_query_4.sql]: This seems incomplete at the moment -> you're returning all hosts with more than one listing neighbourhood, but the requirement is to only return a total number of hosts passing the criteria (having listings at more than one location)

**Review by pmyrsi — CHANGES_REQUESTED** [review]

File: Round_1_query_6.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+SELECT
+100/(COUNT(listing_id))*
+(SELECT
+COUNT(listing_id)
+FROM Listings
+WHERE amenities like '%wifi%') as percent_of_wifi
+FROM Listings
```
Comment (pmyrsi) [inline Round_1_query_6.sql]: This query (even if it works) will not return a single number.

**Review by pmyrsi — CHANGES_REQUESTED** [review]

File: Round_1_query_6.sql (hunk 3 above)
Comment (Michi18) [reply Round_1_query_6.sql]: It hasn't completed yet

**Comment by Michi18** [comment]
Queries, which are completed:
2, 3, 4, 5, 6, 8, 9

**Review by pmyrsi — APPROVED** [review]

**Pull request merged**